docs/source/get_started/basics/basics_qss.ipynb

```diff
@@ -186,7 +172,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.10.6"
+   "version": "3.9.16"
```

Comment:
Did we at some point specify which version to use for notebooks? @bharat-thotakura @richrines1

Reply:
I don't recall a discussion specifically on the version standard to use, but I've tried to stick to version 3.9.16 ever since PR https://github.com/Infleqtion/client-superstaq/pull/490/ standardized it across the notebooks.